### ISSUE TITLE
Paging function in linked objects (complex table)

### ISSUE THREAD

**Issue by rl0infeck** [znuny/znuny-feature-requests#53]
**Problem description:**
I'm using ZNUNY 6.5.2 version and I have a lot of CI which have linked thousands tickets (open and closed). 
It is a problem now because when I would like to open CI page I have to wait a few minutes when page open (bacuse all linked tickets propably have to be taken from DB, generate and show in web gui).
 
**Solution:**
is it possible to add paging feature in CI section like it is in other places (for examples in dashboard - "Shown Tickets" per page) with limit how many linked tickets should appear in one page). Next quastion, is it possible to have the same settings which are for linked tickets in tickets view which give possibility to show only tickets with statuses which we want ?

LinkObject::IgnoreLinkedTicketStateTypes - now this settings doesn't work in CMDB in CI page. This settings works only in Tickets view - AgentTicketZoom)
 
Kind Regards,
rl0infeck

**Comment by rl0infeck** [comment]
![image](https://github.com/znuny/znuny-feature-requests/assets/57048789/00bd15f9-19f7-4c0f-bd9a-ad2ffcc50834)

**Comment by dignin** [comment]
I've added this to an internal Issue/MR ~~for the ITSMConfigurationManagement package~~.

**Comment by rkaldung** [comment]
Internal issue 654

**Comment by rl0infeck** [comment]
Any dates ?

**Comment by hanneshal** [comment]
No, we created the internal issue and it will be checked, effort will be estimated and then it will be planned. 
We post updates here, if there are any.

**Comment by rkaldung** [comment]
Internal issue 654
